static/app/components/core/slider/index.chonk.tsx

```diff
@@ -0,0 +1,317 @@
+import {
+  type CSSProperties,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import type {DO_NOT_USE_ChonkTheme} from '@emotion/react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+import {mergeRefs} from '@react-aria/utils';
+
+import {space} from 'sentry/styles/space';
+
+import type {SliderProps} from './index';
+
+const passthrough = (n: number | '') => n;
+
+export function Slider({
+  formatLabel = passthrough,
+  ref,
+  ...props
+}: SliderProps & {ref: React.Ref<HTMLInputElement>}) {
```

Comment:
```suggestion
}: SliderProps & {ref?: React.Ref<HTMLInputElement>}) {
```
I think the ref is optional?